cog-jq

```diff
@@ -1,6 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -e
 
 # Set the cog argument as the filter (default: '.')
-exec /usr/bin/jq "$COG_ARGV_0"
+RESULT=$(/usr/bin/jq "$COG_ARGV_0")
+
+# Emit JSON if the output sorta looks like JSON
+if [[ "$RESULT" == "{"* ]] || [[ "$RESULT" == "["* ]]; then
```

Comment:
@vanstee Awesome. Thanks for this PR!

Seems to me like we can assume the output is always JSON an`echo 'JSON'` on line 5, without installing `bash`, parsing the result, etc.

(If this plugin ever supports the `--raw-ouput/-r` argument for `jq`, then we could check that to determine whether to `echo JSON`)